client-tmp/angular/src/app/app.component.html

```diff
@@ -0,0 +1,3 @@
+<zero-design-system-provider>
```

Comment:
Fyi, non-Angular version switched to Rapid from Zero. Can be done in future PRs.

Reply:
I changed it in the [next PR](https://github.com/genesiscommunitysuccess/blank-app-seed/pull/253/files)